documents/knowledge/elements.yaml

```diff
@@ -4488,6 +4488,24 @@
   tests:
     - "[[1,5,3],[4,2,6]] : [1, 2, 3]"
 
+- element: "Þs"
+  name: All Slices of a List
```

Comment:
I feel like this would make a good overload of a one-byter. Are there any `lst-int`s available?